Lista.h

```diff
@@ -134,7 +134,7 @@ template<class Dato>
 unsigned Lista<Dato>::obtenerPosicion(Dato d)
 {
     bool estaEnLista  = false;
-    unsigned i = 1 ,posicion = -1 ;
+    unsigned i = 0 ,posicion = -1 ;
 
     while( i <= this->tam && !estaEnLista)
```

Comment:
Ya no recorremos hasta tam inclusive
```suggestion
    while( i < this->tam && !estaEnLista)
```

Reply:
Postre